lib/tasks/npm-install.js

```diff
@@ -24,35 +24,38 @@ module.exports = Task.extend({
     };
     var packages = options.packages || [];
 
-    return new Promise(function(resolve, reject) {
+    // npm otherwise is otherwise noisy, already submitted PR for npm to fix
+    // misplaced console.log
+    this.disableLogger();
 
-      // npm otherwise is otherwise noisy, already submitted PR for npm to fix
-      // misplaced console.log
-      this.disableLogger();
+    var load = Promise.denodeify(this.npm.load);
 
-      this.npm.load(npmOptions, function(err) {
-        if (err) {
-          reject(err);
-        } else {
-          this.npm.commands.install(packages, function(err, data) {
-            if (err) {
-              reject(err);
-            } else {
-              resolve(data);
-            }
-          });
-        }
-      }.bind(this));
+    return load(npmOptions)
+      .then(function() {
+        // if install is denodeified outside load.then(),
+        // it throws "Call npm.load(config, cb) before using this command."
+        var install = Promise.denodeify(this.npm.commands.install);
 
-    }.bind(this), 'npm install')
+        return install(packages);
+      }.bind(this))
       .finally(this.finally.bind(this))
       .then(this.announceCompletion.bind(this));
   },
 
-  announceCompletion: function() {
+  announceCompletion: function(data) {
+    this.printPackageNames(data);
```

Comment:
this actually causes a huge amount of mess for initial installations.

running 

```
ember addon my-cool-addon
```

results in 
```
 % ember addon omg-my-addon                                                                                                                                                                                 !6386
version: 0.2.0-master-0a980ba4a5
installing
  create .bowerrc
  create .editorconfig
  create .ember-cli
  create .jshintrc
  create .travis.yml
  create Brocfile.js
  create README.md
  create tests/dummy/app/app.js
  create tests/dummy/app/components/.gitkeep
  create tests/dummy/app/controllers/.gitkeep
  create tests/dummy/app/helpers/.gitkeep
  create tests/dummy/app/index.html
  create tests/dummy/app/models/.gitkeep
  create tests/dummy/app/router.js
  create tests/dummy/app/routes/.gitkeep
  create tests/dummy/app/styles/app.css
  create tests/dummy/app/templates/application.hbs
  create tests/dummy/app/templates/components/.gitkeep
  create tests/dummy/app/views/.gitkeep
  create bower.json
  create tests/dummy/config/environment.js
  create .gitignore
  create package.json
  create tests/dummy/public/crossdomain.xml
  create tests/dummy/public/robots.txt
  create testem.json
  create tests/.jshintrc
  create tests/helpers/resolver.js
  create tests/helpers/start-app.js
  create tests/index.html
  create tests/test-helper.js
  create tests/unit/.gitkeep
  create vendor/.gitkeep
  create LICENSE.md
  create addon/.gitkeep
  create config/environment.js
  create app/.gitkeep
  create index.js
  create .npmignore
ember-cli-inject-live-reload@1.3.1 node_modules/ember-cli-inject-live-reload
ember-data@1.0.0-beta.16 node_modules/ember-data
ember-export-application-global@1.0.2 node_modules/ember-export-application-global
ic-ajax@2.0.2 node_modules/ember-cli-ic-ajax/node_modules/ic-ajax
ember-cli-ic-ajax@0.1.1 node_modules/ember-cli-ic-ajax
git-repo-info@1.0.4 node_modules/ember-cli-app-version/node_modules/git-repo-version/node_modules/git-repo-info
git-repo-version@0.2.0 node_modules/ember-cli-app-version/node_modules/git-repo-version
ember-cli-app-version@0.3.3 node_modules/ember-cli-app-version
semver@4.3.1 node_modules/ember-cli-dependency-checker/node_modules/semver
ansi-styles@1.1.0 node_modules/ember-cli-dependency-checker/node_modules/chalk/node_modules/ansi-styles
escape-string-regexp@1.0.3 node_modules/ember-cli-dependency-checker/node_modules/chalk/node_modules/escape-string-regexp
supports-color@0.2.0 node_modules/ember-cli-dependency-checker/node_modules/chalk/node_modules/supports-color
ansi-regex@0.2.1 node_modules/ember-cli-dependency-checker/node_modules/chalk/node_modules/has-ansi/node_modules/ansi-regex
has-ansi@0.1.0 node_modules/ember-cli-dependency-checker/node_modules/chalk/node_modules/has-ansi
ansi-regex@0.2.1 node_modules/ember-cli-dependency-checker/node_modules/chalk/node_modules/strip-ansi/node_modules/ansi-regex
strip-ansi@0.3.0 node_modules/ember-cli-dependency-checker/node_modules/chalk/node_modules/strip-ansi
chalk@0.5.1 node_modules/ember-cli-dependency-checker/node_modules/chalk
ember-cli-dependency-checker@0.0.8 node_modules/ember-cli-dependency-checker
content-type@1.0.1 node_modules/ember-cli-content-security-policy/node_modules/body-parser/node_modules/content-type
bytes@1.0.0 node_modules/ember-cli-content-security-policy/node_modules/body-parser/node_modules/bytes
raw-body@1.3.3 node_modules/ember-cli-content-security-policy/node_modules/body-parser/node_modules/raw-body
depd@1.0.0 node_modules/ember-cli-content-security-policy/node_modules/body-parser/node_modules/depd
qs@2.4.1 node_modules/ember-cli-content-security-policy/node_modules/body-parser/node_modules/qs
ee-first@1.1.0 node_modules/ember-cli-content-security-policy/node_modules/body-parser/node_modules/on-finished/node_modules/ee-first
on-finished@2.2.0 node_modules/ember-cli-content-security-policy/node_modules/body-parser/node_modules/on-finished
ms@0.7.0 node_modules/ember-cli-content-security-policy/node_modules/body-parser/node_modules/debug/node_modules/ms
debug@2.1.3 node_modules/ember-cli-content-security-policy/node_modules/body-parser/node_modules/debug
iconv-lite@0.4.7 node_modules/ember-cli-content-security-policy/node_modules/body-parser/node_modules/iconv-lite
media-typer@0.3.0 node_modules/ember-cli-content-security-policy/node_modules/body-parser/node_modules/type-is/node_modules/media-typer
mime-db@1.8.0 node_modules/ember-cli-content-security-policy/node_modules/body-parser/node_modules/type-is/node_modules/mime-types/node_modules/mime-db
mime-types@2.0.10 node_modules/ember-cli-content-security-policy/node_modules/body-parser/node_modules/type-is/node_modules/mime-types
type-is@1.6.1 node_modules/ember-cli-content-security-policy/node_modules/body-parser/node_modules/type-is
body-parser@1.12.2 node_modules/ember-cli-content-security-policy/node_modules/body-parser
ember-cli-content-security-policy@0.4.0 node_modules/ember-cli-content-security-policy
broccoli-asset-rewrite@1.0.4 node_modules/broccoli-asset-rev/node_modules/broccoli-asset-rewrite
rsvp@3.0.17 node_modules/broccoli-asset-rev/node_modules/rsvp
broccoli-writer@0.1.1 node_modules/broccoli-asset-rev/node_modules/broccoli-filter/node_modules/broccoli-writer
promise-map-series@0.2.1 node_modules/broccoli-asset-rev/node_modules/broccoli-filter/node_modules/promise-map-series
walk-sync@0.1.3 node_modules/broccoli-asset-rev/node_modules/broccoli-filter/node_modules/walk-sync
mkdirp@0.3.5 node_modules/broccoli-asset-rev/node_modules/broccoli-filter/node_modules/mkdirp
inherits@2.0.1 node_modules/broccoli-asset-rev/node_modules/broccoli-filter/node_modules/broccoli-kitchen-sink-helpers/node_modules/glob/node_modules/inherits
graceful-fs@3.0.6 node_modules/broccoli-asset-rev/node_modules/broccoli-filter/node_modules/broccoli-kitchen-sink-helpers/node_modules/glob/node_modules/graceful-fs
wrappy@1.0.1 node_modules/broccoli-asset-rev/node_modules/broccoli-filter/node_modules/broccoli-kitchen-sink-helpers/node_modules/glob/node_modules/once/node_modules/wrappy
once@1.3.1 node_modules/broccoli-asset-rev/node_modules/broccoli-filter/node_modules/broccoli-kitchen-sink-helpers/node_modules/glob/node_modules/once
sigmund@1.0.0 node_modules/broccoli-asset-rev/node_modules/broccoli-filter/node_modules/broccoli-kitchen-sink-helpers/node_modules/glob/node_modules/minimatch/node_modules/sigmund
lru-cache@2.5.0 node_modules/broccoli-asset-rev/node_modules/broccoli-filter/node_modules/broccoli-kitchen-sink-helpers/node_modules/glob/node_modules/minimatch/node_modules/lru-cache
minimatch@0.3.0 node_modules/broccoli-asset-rev/node_modules/broccoli-filter/node_modules/broccoli-kitchen-sink-helpers/node_modules/glob/node_modules/minimatch
glob@4.0.4 node_modules/broccoli-asset-rev/node_modules/broccoli-filter/node_modules/broccoli-kitchen-sink-helpers/node_modules/glob
broccoli-kitchen-sink-helpers@0.2.6 node_modules/broccoli-asset-rev/node_modules/broccoli-filter/node_modules/broccoli-kitchen-sink-helpers
mktemp@0.3.5 node_modules/broccoli-asset-rev/node_modules/broccoli-filter/node_modules/quick-temp/node_modules/mktemp
rimraf@2.2.8 node_modules/broccoli-asset-rev/node_modules/broccoli-filter/node_modules/quick-temp/node_modules/rimraf
underscore.string@2.3.3 node_modules/broccoli-asset-rev/node_modules/broccoli-filter/node_modules/quick-temp/node_modules/underscore.string
quick-temp@0.1.2 node_modules/broccoli-asset-rev/node_modules/broccoli-filter/node_modules/quick-temp
broccoli-filter@0.1.12 node_modules/broccoli-asset-rev/node_modules/broccoli-filter
broccoli-asset-rev@2.0.3 node_modules/broccoli-asset-rev
semver@4.3.1 node_modules/ember-cli-htmlbars/node_modules/ember-cli-version-checker/node_modules/semver
ember-cli-version-checker@1.0.2 node_modules/ember-cli-htmlbars/node_modules/ember-cli-version-checker
broccoli-writer@0.1.1 node_modules/ember-cli-htmlbars/node_modules/broccoli-filter/node_modules/broccoli-writer
promise-map-series@0.2.1 node_modules/ember-cli-htmlbars/node_modules/broccoli-filter/node_modules/promise-map-series
walk-sync@0.1.3 node_modules/ember-cli-htmlbars/node_modules/broccoli-filter/node_modules/walk-sync
mkdirp@0.3.5 node_modules/ember-cli-htmlbars/node_modules/broccoli-filter/node_modules/mkdirp
inherits@2.0.1 node_modules/ember-cli-htmlbars/node_modules/broccoli-filter/node_modules/broccoli-kitchen-sink-helpers/node_modules/glob/node_modules/inherits
graceful-fs@3.0.6 node_modules/ember-cli-htmlbars/node_modules/broccoli-filter/node_modules/broccoli-kitchen-sink-helpers/node_modules/glob/node_modules/graceful-fs
wrappy@1.0.1 node_modules/ember-cli-htmlbars/node_modules/broccoli-filter/node_modules/broccoli-kitchen-sink-helpers/node_modules/glob/node_modules/once/node_modules/wrappy
once@1.3.1 node_modules/ember-cli-htmlbars/node_modules/broccoli-filter/node_modules/broccoli-kitchen-sink-helpers/node_modules/glob/node_modules/once
sigmund@1.0.0 node_modules/ember-cli-htmlbars/node_modules/broccoli-filter/node_modules/broccoli-kitchen-sink-helpers/node_modules/glob/node_modules/minimatch/node_modules/sigmund
lru-cache@2.5.0 node_modules/ember-cli-htmlbars/node_modules/broccoli-filter/node_modules/broccoli-kitchen-sink-helpers/node_modules/glob/node_modules/minimatch/node_modules/lru-cache
minimatch@0.3.0 node_modules/ember-cli-htmlbars/node_modules/broccoli-filter/node_modules/broccoli-kitchen-sink-helpers/node_modules/glob/node_modules/minimatch
glob@4.0.4 node_modules/ember-cli-htmlbars/node_modules/broccoli-filter/node_modules/broccoli-kitchen-sink-helpers/node_modules/glob
broccoli-kitchen-sink-helpers@0.2.6 node_modules/ember-cli-htmlbars/node_modules/broccoli-filter/node_modules/broccoli-kitchen-sink-helpers
rsvp@3.0.17 node_modules/ember-cli-htmlbars/node_modules/broccoli-filter/node_modules/rsvp
mktemp@0.3.5 node_modules/ember-cli-htmlbars/node_modules/broccoli-filter/node_modules/quick-temp/node_modules/mktemp
rimraf@2.2.8 node_modules/ember-cli-htmlbars/node_modules/broccoli-filter/node_modules/quick-temp/node_modules/rimraf
underscore.string@2.3.3 node_modules/ember-cli-htmlbars/node_modules/broccoli-filter/node_modules/quick-temp/node_modules/underscore.string
quick-temp@0.1.2 node_modules/ember-cli-htmlbars/node_modules/broccoli-filter/node_modules/quick-temp
broccoli-filter@0.1.12 node_modules/ember-cli-htmlbars/node_modules/broccoli-filter
ember-cli-htmlbars@0.7.4 node_modules/ember-cli-htmlbars
has-color@0.1.7 node_modules/ember-cli-qunit/node_modules/broccoli-jshint/node_modules/chalk/node_modules/has-color
ansi-styles@1.0.0 node_modules/ember-cli-qunit/node_modules/broccoli-jshint/node_modules/chalk/node_modules/ansi-styles
strip-ansi@0.1.1 node_modules/ember-cli-qunit/node_modules/broccoli-jshint/node_modules/chalk/node_modules/strip-ansi
chalk@0.4.0 node_modules/ember-cli-qunit/node_modules/broccoli-jshint/node_modules/chalk
minimist@0.0.8 node_modules/ember-cli-qunit/node_modules/broccoli-jshint/node_modules/mkdirp/node_modules/minimist
mkdirp@0.4.2 node_modules/ember-cli-qunit/node_modules/broccoli-jshint/node_modules/mkdirp
inherits@2.0.1 node_modules/ember-cli-qunit/node_modules/broccoli-jshint/node_modules/findup-sync/node_modules/glob/node_modules/inherits
sigmund@1.0.0 node_modules/ember-cli-qunit/node_modules/broccoli-jshint/node_modules/findup-sync/node_modules/glob/node_modules/minimatch/node_modules/sigmund
lru-cache@2.5.0 node_modules/ember-cli-qunit/node_modules/broccoli-jshint/node_modules/findup-sync/node_modules/glob/node_modules/minimatch/node_modules/lru-cache
minimatch@0.3.0 node_modules/ember-cli-qunit/node_modules/broccoli-jshint/node_modules/findup-sync/node_modules/glob/node_modules/minimatch
glob@3.2.11 node_modules/ember-cli-qunit/node_modules/broccoli-jshint/node_modules/findup-sync/node_modules/glob
lodash@2.4.1 node_modules/ember-cli-qunit/node_modules/broccoli-jshint/node_modules/findup-sync/node_modules/lodash
findup-sync@0.1.3 node_modules/ember-cli-qunit/node_modules/broccoli-jshint/node_modules/findup-sync
strip-json-comments@1.0.2 node_modules/ember-cli-qunit/node_modules/broccoli-jshint/node_modules/jshint/node_modules/strip-json-comments
underscore@1.6.0 node_modules/ember-cli-qunit/node_modules/broccoli-jshint/node_modules/jshint/node_modules/underscore
exit@0.1.2 node_modules/ember-cli-qunit/node_modules/broccoli-jshint/node_modules/jshint/node_modules/exit
date-now@0.1.4 node_modules/ember-cli-qunit/node_modules/broccoli-jshint/node_modules/jshint/node_modules/console-browserify/node_modules/date-now
console-browserify@1.1.0 node_modules/ember-cli-qunit/node_modules/broccoli-jshint/node_modules/jshint/node_modules/console-browserify
sigmund@1.0.0 node_modules/ember-cli-qunit/node_modules/broccoli-jshint/node_modules/jshint/node_modules/minimatch/node_modules/sigmund
lru-cache@2.5.0 node_modules/ember-cli-qunit/node_modules/broccoli-jshint/node_modules/jshint/node_modules/minimatch/node_modules/lru-cache
minimatch@1.0.0 node_modules/ember-cli-qunit/node_modules/broccoli-jshint/node_modules/jshint/node_modules/minimatch
shelljs@0.3.0 node_modules/ember-cli-qunit/node_modules/broccoli-jshint/node_modules/jshint/node_modules/shelljs
inherits@2.0.1 node_modules/ember-cli-qunit/node_modules/broccoli-jshint/node_modules/jshint/node_modules/cli/node_modules/glob/node_modules/inherits
sigmund@1.0.0 node_modules/ember-cli-qunit/node_modules/broccoli-jshint/node_modules/jshint/node_modules/cli/node_modules/glob/node_modules/minimatch/node_modules/sigmund
lru-cache@2.5.0 node_modules/ember-cli-qunit/node_modules/broccoli-jshint/node_modules/jshint/node_modules/cli/node_modules/glob/node_modules/minimatch/node_modules/lru-cache
minimatch@0.3.0 node_modules/ember-cli-qunit/node_modules/broccoli-jshint/node_modules/jshint/node_modules/cli/node_modules/glob/node_modules/minimatch
glob@3.2.11 node_modules/ember-cli-qunit/node_modules/broccoli-jshint/node_modules/jshint/node_modules/cli/node_modules/glob
cli@0.6.5 node_modules/ember-cli-qunit/node_modules/broccoli-jshint/node_modules/jshint/node_modules/cli
domelementtype@1.3.0 node_modules/ember-cli-qunit/node_modules/broccoli-jshint/node_modules/jshint/node_modules/htmlparser2/node_modules/domelementtype
entities@1.0.0 node_modules/ember-cli-qunit/node_modules/broccoli-jshint/node_modules/jshint/node_modules/htmlparser2/node_modules/entities
domhandler@2.3.0 node_modules/ember-cli-qunit/node_modules/broccoli-jshint/node_modules/jshint/node_modules/htmlparser2/node_modules/domhandler
isarray@0.0.1 node_modules/ember-cli-qunit/node_modules/broccoli-jshint/node_modules/jshint/node_modules/htmlparser2/node_modules/readable-stream/node_modules/isarray
inherits@2.0.1 node_modules/ember-cli-qunit/node_modules/broccoli-jshint/node_modules/jshint/node_modules/htmlparser2/node_modules/readable-stream/node_modules/inherits
string_decoder@0.10.31 node_modules/ember-cli-qunit/node_modules/broccoli-jshint/node_modules/jshint/node_modules/htmlparser2/node_modules/readable-stream/node_modules/string_decoder
core-util-is@1.0.1 node_modules/ember-cli-qunit/node_modules/broccoli-jshint/node_modules/jshint/node_modules/htmlparser2/node_modules/readable-stream/node_modules/core-util-is
readable-stream@1.1.13 node_modules/ember-cli-qunit/node_modules/broccoli-jshint/node_modules/jshint/node_modules/htmlparser2/node_modules/readable-stream
domelementtype@1.1.3 node_modules/ember-cli-qunit/node_modules/broccoli-jshint/node_modules/jshint/node_modules/htmlparser2/node_modules/domutils/node_modules/dom-serializer/node_modules/domelementtype
entities@1.1.1 node_modules/ember-cli-qunit/node_modules/broccoli-jshint/node_modules/jshint/node_modules/htmlparser2/node_modules/domutils/node_modules/dom-serializer/node_modules/entities
dom-serializer@0.1.0 node_modules/ember-cli-qunit/node_modules/broccoli-jshint/node_modules/jshint/node_modules/htmlparser2/node_modules/domutils/node_modules/dom-serializer
domutils@1.5.1 node_modules/ember-cli-qunit/node_modules/broccoli-jshint/node_modules/jshint/node_modules/htmlparser2/node_modules/domutils
htmlparser2@3.8.2 node_modules/ember-cli-qunit/node_modules/broccoli-jshint/node_modules/jshint/node_modules/htmlparser2
jshint@2.6.3 node_modules/ember-cli-qunit/node_modules/broccoli-jshint/node_modules/jshint
broccoli-writer@0.1.1 node_modules/ember-cli-qunit/node_modules/broccoli-jshint/node_modules/broccoli-filter/node_modules/broccoli-writer
promise-map-series@0.2.1 node_modules/ember-cli-qunit/node_modules/broccoli-jshint/node_modules/broccoli-filter/node_modules/promise-map-series
walk-sync@0.1.3 node_modules/ember-cli-qunit/node_modules/broccoli-jshint/node_modules/broccoli-filter/node_modules/walk-sync
mkdirp@0.3.5 node_modules/ember-cli-qunit/node_modules/broccoli-jshint/node_modules/broccoli-filter/node_modules/mkdirp
inherits@2.0.1 node_modules/ember-cli-qunit/node_modules/broccoli-jshint/node_modules/broccoli-filter/node_modules/broccoli-kitchen-sink-helpers/node_modules/glob/node_modules/inherits
graceful-fs@3.0.6 node_modules/ember-cli-qunit/node_modules/broccoli-jshint/node_modules/broccoli-filter/node_modules/broccoli-kitchen-sink-helpers/node_modules/glob/node_modules/graceful-fs
wrappy@1.0.1 node_modules/ember-cli-qunit/node_modules/broccoli-jshint/node_modules/broccoli-filter/node_modules/broccoli-kitchen-sink-helpers/node_modules/glob/node_modules/once/node_modules/wrappy
once@1.3.1 node_modules/ember-cli-qunit/node_modules/broccoli-jshint/node_modules/broccoli-filter/node_modules/broccoli-kitchen-sink-helpers/node_modules/glob/node_modules/once
sigmund@1.0.0 node_modules/ember-cli-qunit/node_modules/broccoli-jshint/node_modules/broccoli-filter/node_modules/broccoli-kitchen-sink-helpers/node_modules/glob/node_modules/minimatch/node_modules/sigmund
lru-cache@2.5.0 node_modules/ember-cli-qunit/node_modules/broccoli-jshint/node_modules/broccoli-filter/node_modules/broccoli-kitchen-sink-helpers/node_modules/glob/node_modules/minimatch/node_modules/lru-cache
minimatch@0.3.0 node_modules/ember-cli-qunit/node_modules/broccoli-jshint/node_modules/broccoli-filter/node_modules/broccoli-kitchen-sink-helpers/node_modules/glob/node_modules/minimatch
glob@4.0.4 node_modules/ember-cli-qunit/node_modules/broccoli-jshint/node_modules/broccoli-filter/node_modules/broccoli-kitchen-sink-helpers/node_modules/glob
broccoli-kitchen-sink-helpers@0.2.6 node_modules/ember-cli-qunit/node_modules/broccoli-jshint/node_modules/broccoli-filter/node_modules/broccoli-kitchen-sink-helpers
rsvp@3.0.17 node_modules/ember-cli-qunit/node_modules/broccoli-jshint/node_modules/broccoli-filter/node_modules/rsvp
mktemp@0.3.5 node_modules/ember-cli-qunit/node_modules/broccoli-jshint/node_modules/broccoli-filter/node_modules/quick-temp/node_modules/mktemp
rimraf@2.2.8 node_modules/ember-cli-qunit/node_modules/broccoli-jshint/node_modules/broccoli-filter/node_modules/quick-temp/node_modules/rimraf
underscore.string@2.3.3 node_modules/ember-cli-qunit/node_modules/broccoli-jshint/node_modules/broccoli-filter/node_modules/quick-temp/node_modules/underscore.string
quick-temp@0.1.2 node_modules/ember-cli-qunit/node_modules/broccoli-jshint/node_modules/broccoli-filter/node_modules/quick-temp
broccoli-filter@0.1.12 node_modules/ember-cli-qunit/node_modules/broccoli-jshint/node_modules/broccoli-filter
broccoli-jshint@0.5.5 node_modules/ember-cli-qunit/node_modules/broccoli-jshint
ember-cli-qunit@0.3.9 node_modules/ember-cli-qunit
semver@4.3.1 node_modules/ember-cli-babel/node_modules/ember-cli-version-checker/node_modules/semver
ember-cli-version-checker@1.0.2 node_modules/ember-cli-babel/node_modules/ember-cli-version-checker
broccoli-writer@0.1.1 node_modules/ember-cli-babel/node_modules/broccoli-filter/node_modules/broccoli-writer
promise-map-series@0.2.1 node_modules/ember-cli-babel/node_modules/broccoli-filter/node_modules/promise-map-series
walk-sync@0.1.3 node_modules/ember-cli-babel/node_modules/broccoli-filter/node_modules/walk-sync
mkdirp@0.3.5 node_modules/ember-cli-babel/node_modules/broccoli-filter/node_modules/mkdirp
inherits@2.0.1 node_modules/ember-cli-babel/node_modules/broccoli-filter/node_modules/broccoli-kitchen-sink-helpers/node_modules/glob/node_modules/inherits
graceful-fs@3.0.6 node_modules/ember-cli-babel/node_modules/broccoli-filter/node_modules/broccoli-kitchen-sink-helpers/node_modules/glob/node_modules/graceful-fs
wrappy@1.0.1 node_modules/ember-cli-babel/node_modules/broccoli-filter/node_modules/broccoli-kitchen-sink-helpers/node_modules/glob/node_modules/once/node_modules/wrappy
once@1.3.1 node_modules/ember-cli-babel/node_modules/broccoli-filter/node_modules/broccoli-kitchen-sink-helpers/node_modules/glob/node_modules/once
sigmund@1.0.0 node_modules/ember-cli-babel/node_modules/broccoli-filter/node_modules/broccoli-kitchen-sink-helpers/node_modules/glob/node_modules/minimatch/node_modules/sigmund
lru-cache@2.5.0 node_modules/ember-cli-babel/node_modules/broccoli-filter/node_modules/broccoli-kitchen-sink-helpers/node_modules/glob/node_modules/minimatch/node_modules/lru-cache
minimatch@0.3.0 node_modules/ember-cli-babel/node_modules/broccoli-filter/node_modules/broccoli-kitchen-sink-helpers/node_modules/glob/node_modules/minimatch
glob@4.0.4 node_modules/ember-cli-babel/node_modules/broccoli-filter/node_modules/broccoli-kitchen-sink-helpers/node_modules/glob
broccoli-kitchen-sink-helpers@0.2.6 node_modules/ember-cli-babel/node_modules/broccoli-filter/node_modules/broccoli-kitchen-sink-helpers
rsvp@3.0.17 node_modules/ember-cli-babel/node_modules/broccoli-filter/node_modules/rsvp
mktemp@0.3.5 node_modules/ember-cli-babel/node_modules/broccoli-filter/node_modules/quick-temp/node_modules/mktemp
rimraf@2.2.8 node_modules/ember-cli-babel/node_modules/broccoli-filter/node_modules/quick-temp/node_modules/rimraf
underscore.string@2.3.3 node_modules/ember-cli-babel/node_modules/broccoli-filter/node_modules/quick-temp/node_modules/underscore.string
quick-temp@0.1.2 node_modules/ember-cli-babel/node_modules/broccoli-filter/node_modules/quick-temp
broccoli-filter@0.1.12 node_modules/ember-cli-babel/node_modules/broccoli-filter
clone@0.2.0 node_modules/ember-cli-babel/node_modules/broccoli-babel-transpiler/node_modules/clone
slash@1.0.0 node_modules/ember-cli-babel/node_modules/broccoli-babel-transpiler/node_modules/babel-core/node_modules/slash
trim-right@1.0.0 node_modules/ember-cli-babel/node_modules/broccoli-babel-transpiler/node_modules/babel-core/node_modules/trim-right
to-fast-properties@1.0.1 node_modules/ember-cli-babel/node_modules/broccoli-babel-transpiler/node_modules/babel-core/node_modules/to-fast-properties
shebang-regex@1.0.0 node_modules/ember-cli-babel/node_modules/broccoli-babel-transpiler/node_modules/babel-core/node_modules/shebang-regex
path-is-absolute@1.0.0 node_modules/ember-cli-babel/node_modules/broccoli-babel-transpiler/node_modules/babel-core/node_modules/path-is-absolute
fs-readdir-recursive@0.1.1 node_modules/ember-cli-babel/node_modules/broccoli-babel-transpiler/node_modules/babel-core/node_modules/fs-readdir-recursive
leven@1.0.1 node_modules/ember-cli-babel/node_modules/broccoli-babel-transpiler/node_modules/babel-core/node_modules/leven
private@0.1.6 node_modules/ember-cli-babel/node_modules/broccoli-babel-transpiler/node_modules/babel-core/node_modules/private
globals@6.3.0 node_modules/ember-cli-babel/node_modules/broccoli-babel-transpiler/node_modules/babel-core/node_modules/globals
esutils@1.1.6 node_modules/ember-cli-babel/node_modules/broccoli-babel-transpiler/node_modules/babel-core/node_modules/esutils
estraverse@1.9.3 node_modules/ember-cli-babel/node_modules/broccoli-babel-transpiler/node_modules/babel-core/node_modules/estraverse
convert-source-map@0.5.1 node_modules/ember-cli-babel/node_modules/broccoli-babel-transpiler/node_modules/babel-core/node_modules/convert-source-map
left-pad@0.0.3 node_modules/ember-cli-babel/node_modules/broccoli-babel-transpiler/node_modules/babel-core/node_modules/line-numbers/node_modules/left-pad
line-numbers@0.2.0 node_modules/ember-cli-babel/node_modules/broccoli-babel-transpiler/node_modules/babel-core/node_modules/line-numbers
is-finite@1.0.0 node_modules/ember-cli-babel/node_modules/broccoli-babel-transpiler/node_modules/babel-core/node_modules/is-integer/node_modules/is-finite
is-nan@1.0.1 node_modules/ember-cli-babel/node_modules/broccoli-babel-transpiler/node_modules/babel-core/node_modules/is-integer/node_modules/is-nan
is-integer@1.0.4 node_modules/ember-cli-babel/node_modules/broccoli-babel-transpiler/node_modules/babel-core/node_modules/is-integer
js-tokens@1.0.0 node_modules/ember-cli-babel/node_modules/broccoli-babel-transpiler/node_modules/babel-core/node_modules/js-tokens
ms@0.7.0 node_modules/ember-cli-babel/node_modules/broccoli-babel-transpiler/node_modules/babel-core/node_modules/debug/node_modules/ms
debug@2.1.3 node_modules/ember-cli-babel/node_modules/broccoli-babel-transpiler/node_modules/babel-core/node_modules/debug
escape-string-regexp@1.0.3 node_modules/ember-cli-babel/node_modules/broccoli-babel-transpiler/node_modules/babel-core/node_modules/chalk/node_modules/escape-string-regexp
ansi-styles@2.0.1 node_modules/ember-cli-babel/node_modules/broccoli-babel-transpiler/node_modules/babel-core/node_modules/chalk/node_modules/ansi-styles
supports-color@1.3.1 node_modules/ember-cli-babel/node_modules/broccoli-babel-transpiler/node_modules/babel-core/node_modules/chalk/node_modules/supports-color
ansi-regex@1.1.1 node_modules/ember-cli-babel/node_modules/broccoli-babel-transpiler/node_modules/babel-core/node_modules/chalk/node_modules/strip-ansi/node_modules/ansi-regex
strip-ansi@2.0.1 node_modules/ember-cli-babel/node_modules/broccoli-babel-transpiler/node_modules/babel-core/node_modules/chalk/node_modules/strip-ansi
get-stdin@4.0.1 node_modules/ember-cli-babel/node_modules/broccoli-babel-transpiler/node_modules/babel-core/node_modules/chalk/node_modules/has-ansi/node_modules/get-stdin
ansi-regex@1.1.1 node_modules/ember-cli-babel/node_modules/broccoli-babel-transpiler/node_modules/babel-core/node_modules/chalk/node_modules/has-ansi/node_modules/ansi-regex
has-ansi@1.0.3 node_modules/ember-cli-babel/node_modules/broccoli-babel-transpiler/node_modules/babel-core/node_modules/chalk/node_modules/has-ansi
chalk@1.0.0 node_modules/ember-cli-babel/node_modules/broccoli-babel-transpiler/node_modules/babel-core/node_modules/chalk
graceful-readlink@1.0.1 node_modules/ember-cli-babel/node_modules/broccoli-babel-transpiler/node_modules/babel-core/node_modules/commander/node_modules/graceful-readlink
commander@2.7.1 node_modules/ember-cli-babel/node_modules/broccoli-babel-transpiler/node_modules/babel-core/node_modules/commander
get-stdin@4.0.1 node_modules/ember-cli-babel/node_modules/broccoli-babel-transpiler/node_modules/babel-core/node_modules/detect-indent/node_modules/get-stdin
minimist@1.1.1 node_modules/ember-cli-babel/node_modules/broccoli-babel-transpiler/node_modules/babel-core/node_modules/detect-indent/node_modules/minimist
detect-indent@3.0.1 node_modules/ember-cli-babel/node_modules/broccoli-babel-transpiler/node_modules/babel-core/node_modules/detect-indent
ast-types@0.7.0 node_modules/ember-cli-babel/node_modules/broccoli-babel-transpiler/node_modules/babel-core/node_modules/ast-types
is-finite@1.0.0 node_modules/ember-cli-babel/node_modules/broccoli-babel-transpiler/node_modules/babel-core/node_modules/repeating/node_modules/is-finite
object-assign@2.0.0 node_modules/ember-cli-babel/node_modules/broccoli-babel-transpiler/node_modules/babel-core/node_modules/repeating/node_modules/meow/node_modules/object-assign
get-stdin@4.0.1 node_modules/ember-cli-babel/node_modules/broccoli-babel-transpiler/node_modules/babel-core/node_modules/repeating/node_modules/meow/node_modules/indent-string/node_modules/get-stdin
indent-string@1.2.1 node_modules/ember-cli-babel/node_modules/broccoli-babel-transpiler/node_modules/babel-core/node_modules/repeating/node_modules/meow/node_modules/indent-string
map-obj@1.0.0 node_modules/ember-cli-babel/node_modules/broccoli-babel-transpiler/node_modules/babel-core/node_modules/repeating/node_modules/meow/node_modules/camelcase-keys/node_modules/map-obj
camelcase@1.0.2 node_modules/ember-cli-babel/node_modules/broccoli-babel-transpiler/node_modules/babel-core/node_modules/repeating/node_modules/meow/node_modules/camelcase-keys/node_modules/camelcase
camelcase-keys@1.0.0 node_modules/ember-cli-babel/node_modules/broccoli-babel-transpiler/node_modules/babel-core/node_modules/repeating/node_modules/meow/node_modules/camelcase-keys
minimist@1.1.1 node_modules/ember-cli-babel/node_modules/broccoli-babel-transpiler/node_modules/babel-core/node_modules/repeating/node_modules/meow/node_modules/minimist
meow@3.1.0 node_modules/ember-cli-babel/node_modules/broccoli-babel-transpiler/node_modules/babel-core/node_modules/repeating/node_modules/meow
repeating@1.1.2 node_modules/ember-cli-babel/node_modules/broccoli-babel-transpiler/node_modules/babel-core/node_modules/repeating
xtend@4.0.0 node_modules/ember-cli-babel/node_modules/broccoli-babel-transpiler/node_modules/babel-core/node_modules/output-file-sync/node_modules/xtend
minimist@0.0.8 node_modules/ember-cli-babel/node_modules/broccoli-babel-transpiler/node_modules/babel-core/node_modules/output-file-sync/node_modules/mkdirp/node_modules/minimist
mkdirp@0.5.0 node_modules/ember-cli-babel/node_modules/broccoli-babel-transpiler/node_modules/babel-core/node_modules/output-file-sync/node_modules/mkdirp
output-file-sync@1.1.0 node_modules/ember-cli-babel/node_modules/broccoli-babel-transpiler/node_modules/babel-core/node_modules/output-file-sync
acorn-babel@0.11.1-38 node_modules/ember-cli-babel/node_modules/broccoli-babel-transpiler/node_modules/babel-core/node_modules/acorn-babel
amdefine@0.1.0 node_modules/ember-cli-babel/node_modules/broccoli-babel-transpiler/node_modules/babel-core/node_modules/source-map-support/node_modules/source-map/node_modules/amdefine
source-map@0.1.32 node_modules/ember-cli-babel/node_modules/broccoli-babel-transpiler/node_modules/babel-core/node_modules/source-map-support/node_modules/source-map
source-map-support@0.2.10 node_modules/ember-cli-babel/node_modules/broccoli-babel-transpiler/node_modules/babel-core/node_modules/source-map-support
amdefine@0.1.0 node_modules/ember-cli-babel/node_modules/broccoli-babel-transpiler/node_modules/babel-core/node_modules/source-map/node_modules/amdefine
source-map@0.4.2 node_modules/ember-cli-babel/node_modules/broccoli-babel-transpiler/node_modules/babel-core/node_modules/source-map
core-js@0.6.1 node_modules/ember-cli-babel/node_modules/broccoli-babel-transpiler/node_modules/babel-core/node_modules/core-js
regjsgen@0.2.0 node_modules/ember-cli-babel/node_modules/broccoli-babel-transpiler/node_modules/babel-core/node_modules/regexpu/node_modules/regjsgen
regenerate@1.2.1 node_modules/ember-cli-babel/node_modules/broccoli-babel-transpiler/node_modules/babel-core/node_modules/regexpu/node_modules/regenerate
jsesc@0.5.0 node_modules/ember-cli-babel/node_modules/broccoli-babel-transpiler/node_modules/babel-core/node_modules/regexpu/node_modules/regjsparser/node_modules/jsesc
regjsparser@0.1.4 node_modules/ember-cli-babel/node_modules/broccoli-babel-transpiler/node_modules/babel-core/node_modules/regexpu/node_modules/regjsparser
esprima-fb@13001.1.0-dev-harmony-fb node_modules/ember-cli-babel/node_modules/broccoli-babel-transpiler/node_modules/babel-core/node_modules/regexpu/node_modules/recast/node_modules/esprima-fb
recast@0.10.9 node_modules/ember-cli-babel/node_modules/broccoli-babel-transpiler/node_modules/babel-core/node_modules/regexpu/node_modules/recast
regexpu@1.1.2 node_modules/ember-cli-babel/node_modules/broccoli-babel-transpiler/node_modules/babel-core/node_modules/regexpu
through@2.3.6 node_modules/ember-cli-babel/node_modules/broccoli-babel-transpiler/node_modules/babel-core/node_modules/regenerator-babel/node_modules/through
commander@2.5.1 node_modules/ember-cli-babel/node_modules/broccoli-babel-transpiler/node_modules/babel-core/node_modules/regenerator-babel/node_modules/commoner/node_modules/commander
graceful-fs@3.0.6 node_modules/ember-cli-babel/node_modules/broccoli-babel-transpiler/node_modules/babel-core/node_modules/regenerator-babel/node_modules/commoner/node_modules/graceful-fs
install@0.1.8 node_modules/ember-cli-babel/node_modules/broccoli-babel-transpiler/node_modules/babel-core/node_modules/regenerator-babel/node_modules/commoner/node_modules/install
q@1.1.2 node_modules/ember-cli-babel/node_modules/broccoli-babel-transpiler/node_modules/babel-core/node_modules/regenerator-babel/node_modules/commoner/node_modules/q
minimist@0.0.8 node_modules/ember-cli-babel/node_modules/broccoli-babel-transpiler/node_modules/babel-core/node_modules/regenerator-babel/node_modules/commoner/node_modules/mkdirp/node_modules/minimist
mkdirp@0.5.0 node_modules/ember-cli-babel/node_modules/broccoli-babel-transpiler/node_modules/babel-core/node_modules/regenerator-babel/node_modules/commoner/node_modules/mkdirp
inherits@2.0.1 node_modules/ember-cli-babel/node_modules/broccoli-babel-transpiler/node_modules/babel-core/node_modules/regenerator-babel/node_modules/commoner/node_modules/glob/node_modules/inherits
wrappy@1.0.1 node_modules/ember-cli-babel/node_modules/broccoli-babel-transpiler/node_modules/babel-core/node_modules/regenerator-babel/node_modules/commoner/node_modules/glob/node_modules/once/node_modules/wrappy
once@1.3.1 node_modules/ember-cli-babel/node_modules/broccoli-babel-transpiler/node_modules/babel-core/node_modules/regenerator-babel/node_modules/commoner/node_modules/glob/node_modules/once
wrappy@1.0.1 node_modules/ember-cli-babel/node_modules/broccoli-babel-transpiler/node_modules/babel-core/node_modules/regenerator-babel/node_modules/commoner/node_modules/glob/node_modules/inflight/node_modules/wrappy
inflight@1.0.4 node_modules/ember-cli-babel/node_modules/broccoli-babel-transpiler/node_modules/babel-core/node_modules/regenerator-babel/node_modules/commoner/node_modules/glob/node_modules/inflight
sigmund@1.0.0 node_modules/ember-cli-babel/node_modules/broccoli-babel-transpiler/node_modules/babel-core/node_modules/regenerator-babel/node_modules/commoner/node_modules/glob/node_modules/minimatch/node_modules/sigmund
lru-cache@2.5.0 node_modules/ember-cli-babel/node_modules/broccoli-babel-transpiler/node_modules/babel-core/node_modules/regenerator-babel/node_modules/commoner/node_modules/glob/node_modules/minimatch/node_modules/lru-cache
minimatch@1.0.0 node_modules/ember-cli-babel/node_modules/broccoli-babel-transpiler/node_modules/babel-core/node_modules/regenerator-babel/node_modules/commoner/node_modules/glob/node_modules/minimatch
glob@4.2.2 node_modules/ember-cli-babel/node_modules/broccoli-babel-transpiler/node_modules/babel-core/node_modules/regenerator-babel/node_modules/commoner/node_modules/glob
iconv-lite@0.4.7 node_modules/ember-cli-babel/node_modules/broccoli-babel-transpiler/node_modules/babel-core/node_modules/regenerator-babel/node_modules/commoner/node_modules/iconv-lite
ast-types@0.6.16 node_modules/ember-cli-babel/node_modules/broccoli-babel-transpiler/node_modules/babel-core/node_modules/regenerator-babel/node_modules/commoner/node_modules/recast/node_modules/ast-types
amdefine@0.1.0 node_modules/ember-cli-babel/node_modules/broccoli-babel-transpiler/node_modules/babel-core/node_modules/regenerator-babel/node_modules/commoner/node_modules/recast/node_modules/source-map/node_modules/amdefine
source-map@0.1.43 node_modules/ember-cli-babel/node_modules/broccoli-babel-transpiler/node_modules/babel-core/node_modules/regenerator-babel/node_modules/commoner/node_modules/recast/node_modules/source-map
esprima-fb@10001.1.0-dev-harmony-fb node_modules/ember-cli-babel/node_modules/broccoli-babel-transpiler/node_modules/babel-core/node_modules/regenerator-babel/node_modules/commoner/node_modules/recast/node_modules/esprima-fb
recast@0.9.18 node_modules/ember-cli-babel/node_modules/broccoli-babel-transpiler/node_modules/babel-core/node_modules/regenerator-babel/node_modules/commoner/node_modules/recast
commoner@0.10.1 node_modules/ember-cli-babel/node_modules/broccoli-babel-transpiler/node_modules/babel-core/node_modules/regenerator-babel/node_modules/commoner
regenerator-babel@0.8.13-2 node_modules/ember-cli-babel/node_modules/broccoli-babel-transpiler/node_modules/babel-core/node_modules/regenerator-babel
lodash@3.5.0 node_modules/ember-cli-babel/node_modules/broccoli-babel-transpiler/node_modules/babel-core/node_modules/lodash
babel-core@4.7.16 node_modules/ember-cli-babel/node_modules/broccoli-babel-transpiler/node_modules/babel-core
broccoli-babel-transpiler@4.0.1 node_modules/ember-cli-babel/node_modules/broccoli-babel-transpiler
ember-cli-babel@4.3.0 node_modules/ember-cli-babel
source-map-url@0.3.0 node_modules/ember-cli-uglify/node_modules/broccoli-uglify-sourcemap/node_modules/source-map-url
walk-sync@0.1.3 node_modules/ember-cli-uglify/node_modules/broccoli-uglify-sourcemap/node_modules/walk-sync
minimist@0.0.8 node_modules/ember-cli-uglify/node_modules/broccoli-uglify-sourcemap/node_modules/mkdirp/node_modules/minimist
mkdirp@0.5.0 node_modules/ember-cli-uglify/node_modules/broccoli-uglify-sourcemap/node_modules/mkdirp
copy-dereference@1.0.0 node_modules/ember-cli-uglify/node_modules/broccoli-uglify-sourcemap/node_modules/symlink-or-copy/node_modules/copy-dereference
symlink-or-copy@1.0.1 node_modules/ember-cli-uglify/node_modules/broccoli-uglify-sourcemap/node_modules/symlink-or-copy
uglify-to-browserify@1.0.2 node_modules/ember-cli-uglify/node_modules/broccoli-uglify-sourcemap/node_modules/uglify-js/node_modules/uglify-to-browserify
async@0.2.10 node_modules/ember-cli-uglify/node_modules/broccoli-uglify-sourcemap/node_modules/uglify-js/node_modules/async
yargs@1.3.3 node_modules/ember-cli-uglify/node_modules/broccoli-uglify-sourcemap/node_modules/uglify-js/node_modules/yargs
amdefine@0.1.0 node_modules/ember-cli-uglify/node_modules/broccoli-uglify-sourcemap/node_modules/uglify-js/node_modules/source-map/node_modules/amdefine
source-map@0.1.34 node_modules/ember-cli-uglify/node_modules/broccoli-uglify-sourcemap/node_modules/uglify-js/node_modules/source-map
uglify-js@2.4.17 node_modules/ember-cli-uglify/node_modules/broccoli-uglify-sourcemap/node_modules/uglify-js
mktemp@0.3.5 node_modules/ember-cli-uglify/node_modules/broccoli-uglify-sourcemap/node_modules/broccoli-writer/node_modules/quick-temp/node_modules/mktemp
rimraf@2.2.8 node_modules/ember-cli-uglify/node_modules/broccoli-uglify-sourcemap/node_modules/broccoli-writer/node_modules/quick-temp/node_modules/rimraf
underscore.string@2.3.3 node_modules/ember-cli-uglify/node_modules/broccoli-uglify-sourcemap/node_modules/broccoli-writer/node_modules/quick-temp/node_modules/underscore.string
quick-temp@0.1.2 node_modules/ember-cli-uglify/node_modules/broccoli-uglify-sourcemap/node_modules/broccoli-writer/node_modules/quick-temp
rsvp@3.0.17 node_modules/ember-cli-uglify/node_modules/broccoli-uglify-sourcemap/node_modules/broccoli-writer/node_modules/rsvp
broccoli-writer@0.1.1 node_modules/ember-cli-uglify/node_modules/broccoli-uglify-sourcemap/node_modules/broccoli-writer
lodash-node@2.4.1 node_modules/ember-cli-uglify/node_modules/broccoli-uglify-sourcemap/node_modules/lodash-node
broccoli-uglify-sourcemap@0.2.1 node_modules/ember-cli-uglify/node_modules/broccoli-uglify-sourcemap
ember-cli-uglify@1.0.1 node_modules/ember-cli-uglify
abbrev@1.0.5 node_modules/ember-cli/node_modules/abbrev
ember-cli-copy-dereference@1.0.0 node_modules/ember-cli/node_modules/ember-cli-copy-dereference
broccoli-writer@0.1.1 node_modules/ember-cli/node_modules/broccoli-writer
js-string-escape@1.0.0 node_modules/ember-cli/node_modules/js-string-escape
promise-map-series@0.2.1 node_modules/ember-cli/node_modules/promise-map-series
rimraf@2.2.8 node_modules/ember-cli/node_modules/rimraf
temp@0.8.1 node_modules/ember-cli/node_modules/temp
broccoli-caching-writer@0.5.1 node_modules/ember-cli/node_modules/broccoli-caching-writer
marked@0.3.3 node_modules/ember-cli/node_modules/marked
exit@0.1.2 node_modules/ember-cli/node_modules/exit
ms@0.7.0 node_modules/ember-cli/node_modules/debug/node_modules/ms
debug@2.1.3 node_modules/ember-cli/node_modules/debug
copy-dereference@1.0.0 node_modules/ember-cli/node_modules/symlink-or-copy/node_modules/copy-dereference
symlink-or-copy@1.0.1 node_modules/ember-cli/node_modules/symlink-or-copy
requires-port@0.0.0 node_modules/ember-cli/node_modules/http-proxy/node_modules/requires-port
eventemitter3@0.1.6 node_modules/ember-cli/node_modules/http-proxy/node_modules/eventemitter3
http-proxy@1.9.0 node_modules/ember-cli/node_modules/http-proxy
jsonfile@2.0.0 node_modules/ember-cli/node_modules/yam/node_modules/fs-extra/node_modules/jsonfile
graceful-fs@3.0.6 node_modules/ember-cli/node_modules/yam/node_modules/fs-extra/node_modules/graceful-fs
fs-extra@0.16.5 node_modules/ember-cli/node_modules/yam/node_modules/fs-extra
lodash._arraycopy@3.0.0 node_modules/ember-cli/node_modules/yam/node_modules/lodash.merge/node_modules/lodash._arraycopy
lodash._arrayeach@3.0.0 node_modules/ember-cli/node_modules/yam/node_modules/lodash.merge/node_modules/lodash._arrayeach
lodash.isarguments@3.0.0 node_modules/ember-cli/node_modules/yam/node_modules/lodash.merge/node_modules/lodash.isarguments
lodash._basefor@3.0.0 node_modules/ember-cli/node_modules/yam/node_modules/lodash.merge/node_modules/lodash._basefor
lodash.istypedarray@3.0.0 node_modules/ember-cli/node_modules/yam/node_modules/lodash.merge/node_modules/lodash.istypedarray
lodash.isarray@3.0.0 node_modules/ember-cli/node_modules/yam/node_modules/lodash.merge/node_modules/lodash.isarray
lodash._basecopy@3.0.0 node_modules/ember-cli/node_modules/yam/node_modules/lodash.merge/node_modules/lodash.toplainobject/node_modules/lodash._basecopy
lodash.keysin@3.0.3 node_modules/ember-cli/node_modules/yam/node_modules/lodash.merge/node_modules/lodash.toplainobject/node_modules/lodash.keysin
lodash.toplainobject@3.0.0 node_modules/ember-cli/node_modules/yam/node_modules/lodash.merge/node_modules/lodash.toplainobject
lodash.isnative@3.0.0 node_modules/ember-cli/node_modules/yam/node_modules/lodash.merge/node_modules/lodash.keys/node_modules/lodash.isnative
lodash.keys@3.0.4 node_modules/ember-cli/node_modules/yam/node_modules/lodash.merge/node_modules/lodash.keys
lodash.isnative@3.0.0 node_modules/ember-cli/node_modules/yam/node_modules/lodash.merge/node_modules/lodash.isplainobject/node_modules/lodash.isnative
lodash.keysin@3.0.3 node_modules/ember-cli/node_modules/yam/node_modules/lodash.merge/node_modules/lodash.isplainobject/node_modules/lodash.keysin
lodash.isplainobject@3.0.0 node_modules/ember-cli/node_modules/yam/node_modules/lodash.merge/node_modules/lodash.isplainobject
lodash._bindcallback@3.0.0 node_modules/ember-cli/node_modules/yam/node_modules/lodash.merge/node_modules/lodash._createassigner/node_modules/lodash._bindcallback
lodash._isiterateecall@3.0.4 node_modules/ember-cli/node_modules/yam/node_modules/lodash.merge/node_modules/lodash._createassigner/node_modules/lodash._isiterateecall
lodash._createassigner@3.0.1 node_modules/ember-cli/node_modules/yam/node_modules/lodash.merge/node_modules/lodash._createassigner
lodash.merge@3.0.3 node_modules/ember-cli/node_modules/yam/node_modules/lodash.merge
yam@0.0.18 node_modules/ember-cli/node_modules/yam
broccoli-slow-trees@1.0.0 node_modules/ember-cli/node_modules/broccoli-sane-watcher/node_modules/broccoli-slow-trees
watch@0.10.0 node_modules/ember-cli/node_modules/broccoli-sane-watcher/node_modules/sane/node_modules/watch
sigmund@1.0.0 node_modules/ember-cli/node_modules/broccoli-sane-watcher/node_modules/sane/node_modules/minimatch/node_modules/sigmund
lru-cache@2.5.0 node_modules/ember-cli/node_modules/broccoli-sane-watcher/node_modules/sane/node_modules/minimatch/node_modules/lru-cache
minimatch@0.2.14 node_modules/ember-cli/node_modules/broccoli-sane-watcher/node_modules/sane/node_modules/minimatch
nextback@0.1.0 node_modules/ember-cli/node_modules/broccoli-sane-watcher/node_modules/sane/node_modules/fb-watchman/node_modules/nextback
json-stream@0.2.2 node_modules/ember-cli/node_modules/broccoli-sane-watcher/node_modules/sane/node_modules/fb-watchman/node_modules/json-stream
fb-watchman@0.0.0 node_modules/ember-cli/node_modules/broccoli-sane-watcher/node_modules/sane/node_modules/fb-watchman
tmpl@1.0.3 node_modules/ember-cli/node_modules/broccoli-sane-watcher/node_modules/sane/node_modules/walker/node_modules/makeerror/node_modules/tmpl
makeerror@1.0.10 node_modules/ember-cli/node_modules/broccoli-sane-watcher/node_modules/sane/node_modules/walker/node_modules/makeerror
walker@1.0.6 node_modules/ember-cli/node_modules/broccoli-sane-watcher/node_modules/sane/node_modules/walker
sane@1.0.0-rc1 node_modules/ember-cli/node_modules/broccoli-sane-watcher/node_modules/sane
broccoli-sane-watcher@0.1.1 node_modules/ember-cli/node_modules/broccoli-sane-watcher
broccoli-caching-writer@0.5.3 node_modules/ember-cli/node_modules/broccoli-sourcemap-concat/node_modules/broccoli-caching-writer
delayed-stream@0.0.5 node_modules/ember-cli/node_modules/broccoli-sourcemap-concat/node_modules/combined-stream/node_modules/delayed-stream
combined-stream@0.0.7 node_modules/ember-cli/node_modules/broccoli-sourcemap-concat/node_modules/combined-stream
minimist@0.0.8 node_modules/ember-cli/node_modules/broccoli-sourcemap-concat/node_modules/mkdirp/node_modules/minimist
mkdirp@0.5.0 node_modules/ember-cli/node_modules/broccoli-sourcemap-concat/node_modules/mkdirp
source-map-url@0.3.0 node_modules/ember-cli/node_modules/broccoli-sourcemap-concat/node_modules/fast-sourcemap-concat/node_modules/source-map-url
amdefine@0.1.0 node_modules/ember-cli/node_modules/broccoli-sourcemap-concat/node_modules/fast-sourcemap-concat/node_modules/source-map/node_modules/amdefine
source-map@0.1.43 node_modules/ember-cli/node_modules/broccoli-sourcemap-concat/node_modules/fast-sourcemap-concat/node_modules/source-map
fast-sourcemap-concat@0.2.3 node_modules/ember-cli/node_modules/broccoli-sourcemap-concat/node_modules/fast-sourcemap-concat
broccoli-sourcemap-concat@0.4.3 node_modules/ember-cli/node_modules/broccoli-sourcemap-concat
is@2.1.0 node_modules/ember-cli/node_modules/marked-terminal/node_modules/node.extend/node_modules/is
node.extend@1.1.3 node_modules/ember-cli/node_modules/marked-terminal/node_modules/node.extend
colors@1.0.3 node_modules/ember-cli/node_modules/marked-terminal/node_modules/cli-table/node_modules/colors
cli-table@0.3.1 node_modules/ember-cli/node_modules/marked-terminal/node_modules/cli-table
ansicolors@0.2.1 node_modules/ember-cli/node_modules/marked-terminal/node_modules/cardinal/node_modules/ansicolors
esprima-fb@12001.1.0-dev-harmony-fb node_modules/ember-cli/node_modules/marked-terminal/node_modules/cardinal/node_modules/redeyed/node_modules/esprima-fb
redeyed@0.5.0 node_modules/ember-cli/node_modules/marked-terminal/node_modules/cardinal/node_modules/redeyed
cardinal@0.5.0 node_modules/ember-cli/node_modules/marked-terminal/node_modules/cardinal
marked-terminal@1.2.0 node_modules/ember-cli/node_modules/marked-terminal
broccoli-caching-writer@0.5.3 node_modules/ember-cli/node_modules/broccoli-es6modules/node_modules/broccoli-caching-writer
minimist@0.0.8 node_modules/ember-cli/node_modules/broccoli-es6modules/node_modules/mkdirp/node_modules/minimist
mkdirp@0.5.0 node_modules/ember-cli/node_modules/broccoli-es6modules/node_modules/mkdirp
estraverse@1.9.3 node_modules/ember-cli/node_modules/broccoli-es6modules/node_modules/esperanto/node_modules/estraverse
minimist@1.1.1 node_modules/ember-cli/node_modules/broccoli-es6modules/node_modules/esperanto/node_modules/minimist
vlq@0.2.1 node_modules/ember-cli/node_modules/broccoli-es6modules/node_modules/esperanto/node_modules/magic-string/node_modules/vlq
magic-string@0.4.5 node_modules/ember-cli/node_modules/broccoli-es6modules/node_modules/esperanto/node_modules/magic-string
escape-string-regexp@1.0.3 node_modules/ember-cli/node_modules/broccoli-es6modules/node_modules/esperanto/node_modules/chalk/node_modules/escape-string-regexp
ansi-styles@2.0.1 node_modules/ember-cli/node_modules/broccoli-es6modules/node_modules/esperanto/node_modules/chalk/node_modules/ansi-styles
supports-color@1.3.1 node_modules/ember-cli/node_modules/broccoli-es6modules/node_modules/esperanto/node_modules/chalk/node_modules/supports-color
ansi-regex@1.1.1 node_modules/ember-cli/node_modules/broccoli-es6modules/node_modules/esperanto/node_modules/chalk/node_modules/strip-ansi/node_modules/ansi-regex
strip-ansi@2.0.1 node_modules/ember-cli/node_modules/broccoli-es6modules/node_modules/esperanto/node_modules/chalk/node_modules/strip-ansi
get-stdin@4.0.1 node_modules/ember-cli/node_modules/broccoli-es6modules/node_modules/esperanto/node_modules/chalk/node_modules/has-ansi/node_modules/get-stdin
ansi-regex@1.1.1 node_modules/ember-cli/node_modules/broccoli-es6modules/node_modules/esperanto/node_modules/chalk/node_modules/has-ansi/node_modules/ansi-regex
has-ansi@1.0.3 node_modules/ember-cli/node_modules/broccoli-es6modules/node_modules/esperanto/node_modules/chalk/node_modules/has-ansi
chalk@1.0.0 node_modules/ember-cli/node_modules/broccoli-es6modules/node_modules/esperanto/node_modules/chalk
graceful-fs@3.0.6 node_modules/ember-cli/node_modules/broccoli-es6modules/node_modules/esperanto/node_modules/sander/node_modules/graceful-fs
es6-promise@2.0.1 node_modules/ember-cli/node_modules/broccoli-es6modules/node_modules/esperanto/node_modules/sander/node_modules/es6-promise
sander@0.2.2 node_modules/ember-cli/node_modules/broccoli-es6modules/node_modules/esperanto/node_modules/sander
acorn@0.11.0 node_modules/ember-cli/node_modules/broccoli-es6modules/node_modules/esperanto/node_modules/acorn
esperanto@0.6.17 node_modules/ember-cli/node_modules/broccoli-es6modules/node_modules/esperanto
broccoli-es6modules@0.5.1 node_modules/ember-cli/node_modules/broccoli-es6modules
ember-cli@0.2.0 node_modules/ember-cli
Installed packages for tooling via npm.
Installed browser packages via Bower.
Successfully initialized git.
```